app/controllers/conference_registrations_controller.rb

```diff
@@ -20,6 +20,11 @@ def new
       redirect_to edit_conference_conference_registrations_path(@conference.short_title)
     end
 
+    if @conference.registration_limit_exceeded?
```

Comment:
The registration_limit_exceed method is on conference model. IMHO, I think is conference responsibility to check the limit, not the registration model. 

I have hidden the registration button if limit exceded in the home page. 

As soon as you merge my other PR, I change the _registration partial in the conference show.